### PR TITLE
Version collections

### PR DESCRIPTION
**Pull request by carolynvs**
1. Reorganized version collections
   * The site is generated with a single config.yaml, and is dumped to _site
   * Each release is a collection
     * Releases are output to _site/RELEASE
     * Releases know their version, and have a pretty title
1. List all releases in the top nav

### CODE REVIEW

File: _includes/masthead.html [hunk 1]
```diff
@@ -4,18 +4,16 @@
...
         <a class="site-title" href="{{ '/' | absolute_url }}">{{ site.title }}</a>
         <ul class="visible-links">
-          {% for link in site.data.navigation.main %}
-            {% if link.url contains 'http' %}
-              {% assign domain = '' %}
-            {% else %}
-              {% assign domain = site.url | append: site.baseurl %}
+          {% for c in site.collections %}
+            {% if c.version == nil %}
+              {% continue %}
```
Comment (annegentle) [inline _includes/masthead.html:9]: Brilliant! ❇️ ✨

**Comment by carolynvs** [comment]
I just pushed a new commit that updates the docker build script.

**Comment by annegentle** [comment]
Thank you! I'll change those other scripts in another PR. Woo!

**Comment by annegentle** [comment]
Ohhh.... but I guess I want a collection in a collection? So I get the 4.1 version of the install guide in `{{base_path}}/4.1/install-guide`? Hmmm. Got the scripts all cleaned up anyway.

**Comment by annegentle** [comment]
Can I programmatically access the version value in a YAML file and insert it at some point? 💡 🤔

**Pull request merged**